packages/design-tokens/src/scss/generated/_shadows.scss

```diff
@@ -1,8 +1,8 @@
 // Generated Shadows from Supernova. Do not edit manually.
 $focus: 0 0 0 2px #61b3ff99 !default;
-$shadow-100: 0 1px 3px 0 #00000014, 0 1px 6px 0 #0000001a !default;
-$shadow-200: 0 1px 2px 0 #0000000d, 0 2px 6px 0 #0000000d, 0 4px 6px 0 #00000014 !default;
-$shadow-300: 0 0 1px 0 #0000000d, 0 1px 2px 0 #0000000d, 0 4px 8px 0 #0000000f, 0 8px 16px 0 #0000001a !default;
+$shadow-100: 0 1px 6px 0 #0000001a !default;
```

Comment:
There aren't the correct values. 🙈

Reply:
😨

Reply:
Ha, but it is already broken in Supernova, not in the exporter 😀

Reply:
Or it could be in Figma as well.

Reply:
We've agreed on the minification of the shadow tokens. Current values are 👍🏻 .